02-shutdown/main_test.go

```diff
@@ -65,7 +65,7 @@ func TestWaiter(t *testing.T) {
 		// Discard noisy logs
 		logger := log.New(ioutil.Discard, "", log.LstdFlags)
 		go func() {
-			runServer(logger)
+			runServer(logger, "8000")
```

Comment:
Won't this test fail if the environment variable is set to anything else?
```
PORT=9999 test go test ./...
```

Reply:
No, it doesn't fail (I tested locally). But, I switched this to `0` in case something is already listening on that port when running the test (`0` means use a random free port). The exact port doesn't matter when running a test.

Reply:
Great! Ok, thanks!